website/docs/contributing/backend/overview.md

```diff
@@ -0,0 +1,133 @@
+# Backend
+
+The frontend is written in nodejs/typescript. It's written as a REST API following a CSR (controller, service, repository/store) pattern. The following ADRs are defined for the backend:
+
+## ADRs
+
+We have created a set of ADRs to help guide the development of the backend:
+
+* [Naming](./ADR/naming.md)
+* [Preferred export](./ADR/preferred-export.md)
+
+## Requirements
+
+Before developing on this project you will need two things:
+
+- PostgreSQL 10.x or newer
+- Node.js 14.x or newer
+
+```sh
+yarn install
+yarn run start:dev
+```
+
+## PostgreSQL {#postgresql}
+
+To run and develop unleash, you need to have PostgreSQL database (PostgreSQL v10.x or newer) locally.
+
+> Unleash currently also work with PostgreSQL v9.5+, but this might change in a future feature release, and we have stopped running automatic integration tests below PostgreSQL v10.
+Wh
+### Create a local unleash databases in postgres {#create-a-local-unleash-databases-in-postgres}
+
+```bash
+$ psql postgres <<SQL
+CREATE USER unleash_user WITH PASSWORD 'passord';
+CREATE DATABASE unleash;
+GRANT ALL PRIVILEGES ON DATABASE unleash to unleash_user;
+CREATE DATABASE unleash_test;
+GRANT ALL PRIVILEGES ON DATABASE unleash_test to unleash_user;
```

Comment:
The test database also needs to be configured to use UTC due to how we run our e2e tests. 
```suggestion
GRANT ALL PRIVILEGES ON DATABASE unleash_test to unleash_user;
ALTER DATABASE unleash_test SET timezone TO 'UTC';
```